ompi/mca/osc/rdma/osc_rdma_peer.h

```diff
@@ -45,11 +45,14 @@ struct ompi_osc_rdma_peer_t {
     /** peer flags */
     opal_atomic_int32_t flags;
 
-    /** index into BTL array */
-    uint8_t data_btl_index;
-
-    /** index into BTL array */
-    uint8_t state_btl_index;
+    /** btl used for rdma */
+    struct mca_btl_base_module_t *data_btl;
```

Comment:
What's the difference in the size of the ompi_osc_rdma_peer_t structure with this change and how does that compare with the overall size of the structure?  I think it's going to be 12 bytes bigger, but I may be getting the padding / alignment wrong, because I did it in my head.  While 12 bytes isn't much, there is one of these structures per peer, and that can be millions of peers.  Now, I'd argue that this structure is way too big anyway, but ignoring that, I'm pretty sure this was a space optimization and we should think about the implications of that.